code-challenges.js

```diff
@@ -39,6 +79,18 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+//Create a function that removes all the vowels from a string.
+const noVowels = (string) =>{
+// use .split to split string into array
+// use .filter method through the array to filter out vowels
+    return string.split("").filter(value => {
+// return char as long as value is not upper or lowercase letter
+        return value !== "a" && value !== "A" && value !== "e" && value !== "E" &&  value !=="i" && value !== "I" &&  value !=="o" && value !=="O" && value !== "u" && value !== "U"
```

Comment:
This totally works great! To make our code a bit more DRY, we could use:

`value.toLowerCase() !== 'a'`

This will make sure that we don't need to check for both uppercase and lower case.
To take it a bit further, we could delcare value to be lowercase way ahead of time, before your return, to make it even MORE DRY. 

`value = value.toLowerCase()
return value !== 'a' && value !== 'e' ... `

Just something to think about!